index.html

```diff
@@ -1,199 +1,305 @@
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                    </ul>
+                </div>
+            </div>
+
+        </div>
+
+</div>
+    </section>
+    </article>
+    <footer>
+        <div class="data">
+            <div class="comment">
+                <p class="comment">I'm always interested in hearing about new projects, so if you'd like to chat please get
+                in touch.</p>
             </div>
+           <div class="input-section"> <form action="https://formspree.io/f/xqkovozr" method="POST" class="form">
             
-              <div class="design"><img src="images-3/last-design.svg" alt="Design"></div>
-              <p class="email-account">youngehu8@gmail.com</p>
-              <ul class="socials">
-                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
-                <li><a href="#"><img src="images/linkedin.svg" alt="linkedin"></a></li>
-                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
-                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
-                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
-              </ul>
-          
-            </form>
-            <div class="line"></div>
+
+            <div class="fill-in-name">
+                <input class="input_name" type="text" placeholder="First name" name="name" maxlength="30" required />
+                <input class="input_name" type="text" placeholder="Last name" name="name" maxlength="30" required />
             </div>
-          </footer>
-        
-    </body>
-    </html>
+
+            <input class="input_full-name" type="text" placeholder="Full name" name="name" maxlength="30" required />
+            <input class="input_email" type="email" placeholder="Email address" name="email" required="email" />
+            <textarea class="input_msg" placeholder="Hey I'd like to connect to..." name="message" rows="8"
+                maxlength="500" required></textarea>
+            <div class="Get-in-touch-button"><button class="interaction-button" type="submit">Get in touch</button>
+            </div></div>
+        </div>
+
+        <div class="design"><img src="images-3/last-design.svg" alt="Design"></div>
+        <p class="email-account">youngehu8@gmail.com</p>
+        <ul class="socials">
+            <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/linkedin.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+            <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        </ul>
+
+        </form>
+        <div class="line"></div>
+        </div>
+    </footer>
```

Comment:
👍 Great effort in following the design closely! However, I have a few recommendations to align your implementation with the project requirements and design:

- [ ] According to the project requirements, there should be only one input field for the name in the desktop version, similar to the mobile version. Please remove the additional input field and ensure that you're following the requirement.

- [ ] In the desktop version, the button should be left-aligned with the input field, as per the design. Please update your code to achieve this alignment.

- [ ] The email field should be hidden in the desktop version, following the design. Please make the necessary adjustments to hide the email field as intended.

- [ ] In the desktop version, there is a vertical line before the icons. It's crucial to follow the design closely and include this vertical line in your code.

- [ ] Additionally, don't forget to add the background as specified in the design. It will enhance the visual appeal of your project.

- [ ] Finally, pay attention to proper spacing and alignment for all items, ensuring they match the design. Following the design closely will result in a more polished and professional appearance.
![Frame 43](https://github.com/Young-Gaius/portfolio-setup/assets/55840999/3de5a133-8e16-419f-bb52-02b8b76d4c96)
Taking these recommendations into account will ensure that your implementation meets the project requirements and matches the design accurately. Keep up the great work and happy coding! 😄🚀